.github/workflows/macos.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       matrix:
         env:
           - { os: macos-12, xcode: 13.1 }
-          - { os: macos-14, xcode: 14.3.1, LG_VADDR: 39}
```

Comment:
The approach here was to use the lowest XCode available for the runner
It still show 14.3.1
https://github.com/actions/runner-images/blob/main/images/macos/macos-14-Readme.md

Reply:
Unfortunatelly not for all https://github.com/tamatebako/tebako/actions/runs/11792945112/job/32847863897